### DISCUSSION TITLE
Add A diagram and/or image to show how each relation work

### DISCUSSION THREAD

**Discussion by bastianhilton** [directus/directus#20656] [Feature Requests]
### Summary

It would be nice to have a diagram or image to show how each relation work within Directus. Something like what Strapi has would help to understand how to use the relationship mapping of tables upon creation. 

![Screenshot 2023-12-06 162343](https://github.com/directus/directus/assets/16978186/53adb4a7-3e38-42cd-ac26-b9d7194e67e6)

### Motivation

Creating an easier to understand interface helps with non or new developers and helps with the User Experience when using a data platform like Directus.

### Detailed Design

![Screenshot 2023-12-06 162343](https://github.com/directus/directus/assets/16978186/f261aa54-e7af-4322-8b9b-5e89fdc56e7d)

### Must Have:
- A Ease of Use
- B Ability to modify the field after creation

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

The tradeoff would be that the design around relations would have to be adjusted to include maybe a popup/dialog instead of the current accordian style view.

### Alternatives

Include detailed diagrams maybe even interactive ones in the documentation.

### Adoption Strategy

This isn't a breaking change and since its just design based shouldn't cause any actual change to any underlying database or code.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
